pages/bookmarks.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+
+import RecipeList from "@/components/RecipeList";
+
+export default function BookmarkPage({
+  bookmarkedRecipesIDs,
+  onHandleBookmarkedIcon,
+  recipes,
+}) {
+  const StyledHeadline = styled.h1`
+    text-align: center;
+  `;
```

Comment:
please move this styled component out of the Page Component and at the bottom of this file.